app/Http/Controllers/Api/StatisticController.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use App\Http\Controllers\Controller;
+use App\Material;
+use App\Food;
+use App\OrderItem;
+
+class StatisticController extends ApiController
+{
+    /**
+     * Get count of resources.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function countResources()
+    {
+        $modelFiles = \File::files('../app/');
+
+        foreach ($modelFiles as $modelFile) {
+            $fileName = \File::name($modelFile);
+            if ($fileName == 'OrderItem') {
+                continue;
+            }
+            $modelName = 'App\\'.$fileName;
+            $modelInstance = new $modelName;
+            if ($fileName == 'DailyMenu') {
+                $curDate = date('Y-m-d');
+                $counts[$modelInstance->getTable()] = $modelInstance->where('date', '=', $curDate)->count();
+                continue;
+            }
+            $counts[$modelInstance->getTable()] = $modelInstance->count();
+        }
+
+        return response()->json(collect(['data' => $counts])->merge(['success' => true]));
+    }
+
+    /**
+     * Get top of order by table type
+     *
+     * @param string $tableType type of target table
+     *
+     * @return array
+     */
+    public function getTopOrderOf($tableType)
```

Comment:
Please make this function into model `OrderItem `. And name should be `getTrendOrderByModel`.